linux/datadog_wrapper

```diff
@@ -1,59 +1,106 @@
 #!/usr/bin/env sh
 
-CURRENT_DIR=$(pwd)
+main() {
+    # Required to add the AAS metadata
+    export DD_AZURE_APP_SERVICES=1
+
+    CURRENT_DIR=$(pwd)
+    echo "Set application directory as $CURRENT_DIR"
+
+    echo "Setting Datadog environment variables"
+    setEnvVars
+
+    echo "Creating and switching to the Datadog directory at $DD_DIR"
+    mkdir -p "$DD_DIR" && cd "$DD_DIR" || return
+
+    case "$WEBSITE_STACK" in
+        "NODE")
+            echo "Initializing setup for a Node environment"
+            getBinaries
+            setUpNodeEnv;;
+        "DOTNETCORE")
+            echo "Initializing setup for a .NET environment"
+            getBinaries
+            setUpDotnetEnv;;
+        "*")
+            echo "Unsupported runtime. Exiting Datadog startup"
+            return;;
+    esac
```

Comment:
_Technically_ since we've changed the order of the setup here, we're now creating the `DD_DIR` even in the case where its an unsupported runtime. It's not an issue or anything, but it is a regresssion.

Reply:
My thoughts were that a customer would retry the install and this wouldn't be a hinderance, but I see how it is a regression.

Reply:
Oh yeah, totally not an actual problem. Just wanted to point it out.